src/main/java/aquarium/shop/order/OrderDetails.java

```diff
@@ -30,14 +28,18 @@ public long getId() {
         return id;
     }
 
-    //private HashMap<Product, Integer> content = new HashMap<>();
-    //private BigDecimal wholePrice;
+    @OneToMany(cascade = CascadeType.ALL)
+    @JoinColumn(name = "order_id", referencedColumnName = "id")
+    private List<PurchasedProduct> purchasedProducts;
+
+
+    private BigDecimal wholePrice;
 
     public OrderDetails() {
     }
 
     public OrderDetails(DeliveryDetails deliveryDetails, PaymentStatus paymentStatus,
```

Comment:
It would be nice to add **purchasedProducts** and **wholePrice** to constructor to initialize evenrything in one line.